src/patternfly/components/Masthead/masthead.scss

```diff
@@ -147,32 +189,92 @@ $pf-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
   }
 
   .pf-c-toolbar {
-    --pf-c-toolbar--PaddingTop: var(--pf-c-masthead--c-toolbar--PaddingTop);
```

Comment:
Did you mean to remove these? I'm seeing this in the demo from the toolbar's top/bottom padding



<img width="811" alt="Screen Shot 2021-07-12 at 5 30 02 PM" src="https://user-images.githubusercontent.com/35148959/125363746-cfdc6a00-e336-11eb-91fd-efa9bf623953.png">


<img width="555" alt="Screen Shot 2021-07-12 at 5 28 39 PM" src="https://user-images.githubusercontent.com/35148959/125363696-b76c4f80-e336-11eb-9978-e6c0874f0abf.png">

<img width="351" alt="Screen Shot 2021-07-12 at 5 31 12 PM" src="https://user-images.githubusercontent.com/35148959/125363832-f8fcfa80-e336-11eb-9ab4-7159e98bba1a.png">